src/pages/staff/index.tsx

```diff
@@ -1,8 +1,71 @@
+import { trpc } from "@/client/lib/trpc";
 import React from "react";
-import MyParas from "@/components/myParas/MyParas";
+import PersonTable from "@/components/table/table";
+import { Para, ParaHeadCell } from "@/types/global";
 
-function Staff() {
-  return <MyParas />;
-}
+const Staff = () => {
+  const utils = trpc.useContext();
+  const { data: paras, isLoading } = trpc.case_manager.getMyParas.useQuery();
+
+  const createPara = trpc.para.createPara.useMutation({
+    onSuccess: () => utils.case_manager.getMyParas.invalidate(),
+    onSettled: (data, error) => {
+      if (error) console.log(error.message);
+
+      assignParaToCaseManager.mutate({
+        para_id: data?.user_id as string,
+      });
+    },
+  });
+
+  const assignParaToCaseManager = trpc.case_manager.addPara.useMutation({
+    onSuccess: () => utils.case_manager.getMyParas.invalidate(),
+    onError: (error) => console.log(error.message),
+  });
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+
+    createPara.mutate({
+      first_name: data.get("first_name") as string,
+      last_name: data.get("last_name") as string,
+      email: data.get("email") as string,
+    });
```

Comment:
I think this is old code but would recommend
```suggestion
    await createPara.mutateAsync({
      first_name: data.get("first_name") as string,
      last_name: data.get("last_name") as string,
      email: data.get("email") as string,
    });
```
so the form doesn't get reset if there's an error